apisix/schema_def.lua

```diff
@@ -332,6 +332,10 @@ local upstream_schema = {
             description = "enable websocket for request",
             type        = "boolean"
         },
+        labels = {
+            description = "key/value pairs to specify attributes",
+            type = "table"
```

Comment:
I think it should be `object` here.

Reply:
Need to limit the depth to `1`

Reply:
@membphis agreed, nice catch, thanks. I would make changes as the your comments and add one more failure test case after I find the lua way. 😄

Reply:
@membphis PR is updated and labels are limited to string key/value pairs, so fix the depth problem. 😄